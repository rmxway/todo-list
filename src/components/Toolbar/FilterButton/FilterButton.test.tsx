import { render } from '@testing-library/react';
import { FilterButton } from './';

describe('FilterButton', () => {
	it('Render is fine', () => {
		const { container } = render(<FilterButton name="active" title="Active" />);

		expect(container).toMatchSnapshot();
	});
});
