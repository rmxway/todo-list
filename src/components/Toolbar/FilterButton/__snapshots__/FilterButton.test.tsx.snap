// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterButton Render is fine 1`] = `
<div>
  <label
    class="sc-aYaIB eLJtMe"
    for="active"
  >
    <input
      id="active"
      name="filter"
      type="radio"
    />
    Active
  </label>
</div>
`;
